cargo-shuttle/tests/integration/run.rs

```diff
@@ -1,12 +1,23 @@
-use cargo_shuttle::{Args, Command, ProjectArgs, RunArgs, Shuttle};
+use cargo_shuttle::{Command, ProjectArgs, RunArgs, Shuttle, ShuttleArgs};
 use portpicker::pick_unused_port;
 use reqwest::StatusCode;
 use std::{fs::canonicalize, process::exit, time::Duration};
 use tokio::time::sleep;
 
 /// creates a `cargo-shuttle` run instance with some reasonable defaults set.
 async fn cargo_shuttle_run(working_directory: &str, external: bool) -> String {
-    let working_directory = canonicalize(working_directory).unwrap();
+    let working_directory = match canonicalize(working_directory) {
+        Ok(wd) => wd,
+        Err(e) => {
+            // DEBUG CI (no such file): SLEEP AND TRY AGAIN?
+            println!(
+                "Did not find directory: {} !!! because {:?}",
+                working_directory, e
+            );
+            tokio::time::sleep(tokio::time::Duration::from_millis(500)).await;
+            canonicalize(working_directory).unwrap()
```

Comment:
Hmm, a bit strange. Why would `canonicalize` fail on the CI? Would like to see it fail to debug a bit. Can you put back the `expect` (it was an `unwrap` before, but it is a bit more user-friendly to use `expect`)?